.github/workflows/test.yml

```diff
@@ -44,19 +44,19 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
         with:
-          submodules: recursive
+          submodules: true
       ##########################################################################
       # BUILD LOGIN SERVER DOCKER IMAGE ########################################
       ##########################################################################
-      - name: login server | Build `test` image
+      - name: login server | Build `release` image
         run: |
-          docker build --target login_server -t "gradido/login_server:test" -f ./login_server/Dockerfile login_server/
-          docker save "gradido/login_server:test" > /tmp/login_server.tar
-      - name: Upload Artifact
-        uses: actions/upload-artifact@v2
-        with:
-          name: docker-login-server-test
-          path: /tmp/login_server.tar
+          docker build --target release -t "gradido/login_server:release" -f ./login_server/Dockerfile login_server/
+          #docker save "gradido/login_server:test" > /tmp/login_server.tar
+      #- name: Upload Artifact
+       # uses: actions/upload-artifact@v2
+        #with:
+         # name: docker-login-server-test
+          #path: /tmp/login_server.tar
```

Comment:
I don't know for what reason it is uploaded.

Reply:
I don't, with release I build Gradido_LoginServer with Alpine Linux, 
with test I build Gradido_LoginServer_Test with Ubuntu linux because coverage don't worked with Alpine.